assert/doc.go

```diff
@@ -1,5 +1,5 @@
 // Package assert provides a set of comprehensive testing tools for use with the normal Go testing system.
-//
+// All functions in this package return a bool value indicating whether the test has passed.
 // # Example Usage
```

Comment:
Did you check with pkgsite if it was still displaying a header for the example?

Because I would expect the comment to be like this

```suggestion
// All functions in this package return a bool value indicating whether the test has passed.
	//
 // # Example Usage
```

Also please consider using this

```suggestion
//
// # Note
// All functions in this package return a bool value indicating whether the test has passed.
	//
 // # Example Usage
```

Please check for difference by using pkgsite